### PR TITLE
fix tests that broke build due to renaming of parameters

### PR DESCRIPTION
**Pull request by broodingdark**
also set test URLs used for mocket-io

### CODE REVIEW

File: NexmoConversation/Core/REST/BaseURL.swift [hunk 1]
```diff
@@ -13,9 +13,9 @@ internal struct BaseURL {
     
     /// Environment paths
     private enum Path: String {
-        case socket = "socket_url"
-        case rest = "rest_url"
-        case ips = "ips_url"
+        case socket = "http://127.0.0.1:8888"
```
Comment (shams-ahmed) [inline NexmoConversation/Core/REST/BaseURL.swift]: `Path` is used to provide a type-safe key. BaseURL uses this to fetch the value in the target environment variables.

File: NexmoConversationTests/Core/ConversationControllerTest.swift [hunk 2]
```diff
@@ -119,7 +119,7 @@ class ConversationControllerTest: QuickSpec {
             
             var statusResponse: NXMMember.MemberState?
             
-            _ = self.conversationController.create(create, forUser: join.userId)
+            _ = self.conversationController.create(create, with: join.userId)
```
Comment (shams-ahmed) [inline NexmoConversationTests/Core/ConversationControllerTest.swift]: thanks for fixing these issues...

File: NexmoConversation/Core/REST/BaseURL.swift (hunk 1 above)
Comment (broodingdark) [inline NexmoConversation/Core/REST/BaseURL.swift]: OK, I was not aware of you intentions here, will revert this one :) So I basically need to set the aforementioned env vars?

Comment (shams-ahmed) [inline NexmoConversation/Core/REST/BaseURL.swift]: yep

**Comment by broodingdark** [comment]
Updated PR :)

**Pull request merged**